src/lib/ngsi/Scope.cpp

```diff
@@ -97,16 +97,24 @@ int Scope::fill
 )
 {
   Geometry                    geometry;
-  Georel                      georel;
   std::vector<std::string>    pointStringV;
   int                         points;
   std::vector<orion::Point*>  pointV;
 
+  if (apiVersion == "v1")
+  {
+    type = FIWARE_LOCATION;
+  }
+  else
+  {
+    type = FIWARE_LOCATION_V2;
+  }
+
```

Comment:
I think I wrote these 8 lines ..

Now I wish I had done it like this, on **one** line:

    type = (apiVersion == "v1")? FIWARE_LOCATION : FIWARE_LOCATION_V2: